docs/governance/politeia/politeia.md

```diff
@@ -35,11 +35,12 @@ Below is the basic process for submitting and evaluating proposals:
 1. A proposal is submitted.
 2. The proposal is reviewed by Politeia admins. Spam is censored. 
 3. Valid proposals appear publicly on Politeia. They are open for discussion, but voting does not begin immediately, and the proposal can be edited by its owner in response to community feedback. 
-4. The proposal owner authorizes voting to start. An admin then triggers the start of voting.	
-5. The ticket-voting interval of 2016 blocks (~1 week) begins. A snapshot of the live ticket pool is taken at 256 blocks prior to the start of voting. Every ticket in the pool when this snapshot was taken can vote 'Yes' or 'No' on the proposal. Tickets bought after the snapshot cannot vote on the proposal. If a ticket is called to vote on-chain during the ticket-voting interval (to validate blocks or vote on consensus rule changes), it still has until the end of the ticket-voting interval to vote on the proposal. 
-6. When the ticket-voting period ends, the proposal is formally approved or rejected. There is a quorum requirement for a vote to be considered valid: 20% of the eligible tickets must vote 'Yes' or 'No'. The threshold for a proposal to be approved is 60% 'Yes' votes.
-7. When a proposal with a budget and deliverables is approved, work can begin. The proposal owner can submit claims against the budget as deliverables are completed.
-8. Payment claims will be handled manually by Decred Holdings Group (DHG) until such times as this process can be handled by the [Decred Decentralized Autonomous Entity (DAE)](#decentralized-autonomous-entity-dae).
+4. If a proposer owner decides to abandon their proposal (for instance, if comments from the community indicate it is not likely to pass), they can ask the admins to mark their proposal as ‘Abandoned’. Abandoned proposals appear in the ‘Abandoned’ tab on Politeia. They are publicly viewable, but cannot be edited, commented on, or be authorized to vote once abandoned. Admins can mark a proposal as abandoned at any time (for instance, if it is deemed spam) before the proposal owner authorizes voting. Once the proposal owner authorizes voting, it cannot be abandoned.  
```

Comment:
> 4. If a proposer owner decides to abandon their proposal

Is "proposer owner" intended or is that suppose to be "proposal owner"?

> Admins can mark a proposal as abandoned at any time (for instance, if it is deemed spam) before the proposal owner authorizes voting.

Admins aren't suppose to mark a proposal as abandoned if it is spam.  The only time an admin should mark a proposal as abandoned is if the proposal has been inactive for a few weeks.  Spam shouldn't get approved in the first place.  If a proposal author gets their proposal approved, but then edits it to be spam, then the proposal will be censored.  The ability to censor public proposals hasn't been implemented yet.  There is a issue open for it though.